charts/external-dns/values.yaml

```diff
@@ -76,7 +76,7 @@ podLabels: {}
 podAnnotations: {}
 
 # -- (bool) Set this to `false` to [opt out of API credential automounting](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting) for the `Pod`.
-automountServiceAccountToken:
+automountServiceAccountToken:  # @schema type:[boolean, null]; default: null
```

Comment:
Same here, why not set it by default to `true`?

Reply:
Same as above.